### PR TITLE
docs: Point to ODH Dashboard for MOC

### PR DESCRIPTION
**Pull request by tumido**
Points MOC users to the ODH dashboard rather than custom docs and list all the component current status as well.

Merge together with: https://github.com/operate-first/operate-first.github.io/pull/140
Related to: https://github.com/operate-first/SRE/issues/25, https://github.com/operate-first/SRE/issues/24, https://github.com/operate-first/odh/issues/10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,37 @@ This repository contains all the operational and user documentation for running
...
+
+![ODh Dashboard](docs/assets/images/odh-dashboard.png)
+
+## Component status
+
+| Component               | Availability      | Authentication                                                                     |
+| ----------------------- | ----------------- | ---------------------------------------------------------------------------------- |
+| AI-Library              | ❌ (Not deployed) |                                                                                    |
+| Airflow                 | ❌ (Not deployed) |                                                                                    |
+| Argo                    | ✔️                | None (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/39))   |
```
Comment (anishasthana) [inline README.md:21]: Nit: The table columns look a little off :-)

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -4,13 +4,37 @@ This repository contains all the operational and user documentation for running
 
 ## Getting Started
 
-We have the [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC ([Mass Open Cloud](https://massopen.cloud/)) cluster. All user documentation such as the login process for each of the applications deployed can be found [here](https://github.com/operate-first/odh-moc-support/tree/main/docs/user-docs).
+We have the [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC ([Mass Open Cloud](https://massopen.cloud/)) cluster.
```
[the remark this answers is not in the document]
Comment (tumido) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -4,13 +4,37 @@ This repository contains all the operational and user documentation for running
...
+
+ODH homepage on MOC is the [ODH Dashboard](https://odh-dashboard-opf-dashboard.apps.cnv.massopen.cloud/). From there you can access any of the provided application:
```
[the remark this answers is not in the document]
Comment (tumido) [reply README.md]: Fixed :+1:

File: README.md (hunk 1 above)
Comment (tumido) [reply README.md:21]: image

Are you sure it's not your git diff split view? :thinking:

File: README.md [hunk 4]
```diff
@@ -4,13 +4,37 @@ This repository contains all the operational and user documentation for running
...
+## Component status
+
+| Component               | Availability      | Authentication                                                                     |
+| ----------------------- | ----------------- | ---------------------------------------------------------------------------------- |
+| AI-Library              | ❌ (Not deployed) |                                                                                    |
+| Airflow                 | ❌ (Not deployed) |                                                                                    |
+| Argo                    | ✔️                | None (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/39))   |
+| Grafana                 | ✔️                | Custom (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/37)) |
+| Hue                     | ❌ (Stabilizing)  | Custom (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/47)) |
+| JupyterHub              | ✔️                | Login via Openshift                                                                |
```
Comment (hemajv) [inline README.md:24]: should we maybe rephrase it as `Login via OpenShift using your Gmail account`?

File: README.md (hunk 1 above)
Comment (anishasthana) [reply README.md:21]: Hah, I see the problem. My :heavy_check_mark:  is showing up as smaller than the :x:  in the diff.

File: README.md (hunk 4 above)
Comment (anishasthana) [reply README.md:24]: Google account, perhaps?

Comment (tumido) [reply README.md:24]: I think you can use any account provider mapped to Openshift Auth, can't you? Like.. if you have UNI login, you can use that, or not?

Comment (anishasthana) [reply README.md:24]: I'm not actually sure... I thought most of our rbac was tied to things like "<person>@redhat.com" and all that.

Comment (tumido) [reply README.md:24]: yes.. we do.. but we probably don't expect students/uni researchers to do that.

Comment (hemajv) [reply README.md:24]: yeah, so just to avoid confusion between users thinking that they will need to have a separate account, I was thinking we could mention it as something like: `Login via OpenShift -> moc-sso -> your email account`? we could also include screenshots, if that makes it easier

Comment (tumido) [reply README.md:24]: Ultimately, the link they're clicking at each of the service says "Login via Openshift"

image
image

Which leads to the Openshift provider selection - we probably should mention the `moc-sso` here.
image


The path it takes from there is really out of our control. If they use UNI login or email account, that's transparent to us and we really shouldn't care. We can suggest them to select the appropriate login "type" but I think saying "if you don't have an UNI account, click the other button and use Google account " is not very helpful. :smile: It needs better wording. :smile:

Comment (tumido) [reply README.md]: @hema, @anishasthana how do you like this?

Comment (anishasthana) [reply README.md]: Yeah I think that makes sense.

Comment (hemajv) [reply README.md]: yes I think this looks much better! :+1: :smile:

**Review by hemajv — APPROVED** [review]
lgtm :+1:
